spec/lib/redmine/i18n_spec.rb

```diff
@@ -35,43 +35,35 @@ module OpenProject
     let(:format) { "%d/%m/%Y" }
     let(:user) { build_stubbed(:user) }
 
-    after do
-      Time.zone = nil
-    end
+    describe "#format_time_as_date" do
```

Comment:
There are no tests taking time parameter as string. Not sure if it's used that way in our code and how it deals with time strings without timezone information.
Not sure it's worth it though.

Reply:
same for `#format_time`.

Reply:
I removed the option to pass in strings in [ff45d0d](https://github.com/opf/openproject/pull/16567/commits/ff45d0d0a229e27425abc5eec98278839d85b95f). The tests at least are green and from the calls I looked at randomly, no calls are made using strings. But there are a lot of those. I still think it is ok to make that change with the automatic tests not having spotted anything and with the QA still there to catch things.

What is your opinion, @cbliard

Reply:
If passing strings is never used, then yes definitely worth removing it.

And I don't think adding tests was worth it, but I was wondering what the time zone would be when parsing a string to a time.